### ISSUE TITLE
Receiving tx receipt whereas tx is still pending in the node

### ISSUE THREAD

**Issue by jeremyjams** [iExecBlockchainComputing/iexec-core#183]
```2019-02-07 10:03:07.362  INFO 1 --- [pool-2-thread-2] com.iexec.core.chain.DealWatcherService  : Received deal [dealId:0xe839d6f70530c18c7b367d233f66ceb437075a1e6fa341d79a50e0a6c849df91, block:10305964]
2019-02-07 10:03:07.647  INFO 1 --- [pool-2-thread-2] com.iexec.core.task.TaskService          : Add new task [chainDealId:0xe839d6f70530c18c7b367d233f66ceb437075a1e6fa341d79a50e0a6c849df91, taskIndex:0, imageName:registry.hub.docker.com/iexechub/vanityeth:latest, commandLine:ace, trust:1]
2019-02-07 10:03:07.673  INFO 1 --- [pool-2-thread-2] c.i.core.task.listener.TaskListeners     : Received TaskCreatedEvent [chainTaskId:0x03c34103ace93b3cbd84407fa94c8ce712652b340f55d4cd15770ac4de6fb0eb]
2019-02-07 10:03:08.005  INFO 1 --- [pool-4-thread-1] com.iexec.common.chain.ChainUtils        : ETH balance is fine [balance:2.044204929977103366, estimateTxNb:185]
2019-02-07 10:03:08.009  INFO 1 --- [pool-4-thread-1] com.iexec.core.task.TaskService          : UpdateTaskStatus suceeded [chainTaskId:0x03c34103ace93b3cbd84407fa94c8ce712652b340f55d4cd15770ac4de6fb0eb, currentStatus:RECEIVED, newStatus:INITIALIZING]
2019-02-07 10:03:08.009  INFO 1 --- [pool-4-thread-1] com.iexec.core.chain.IexecHubService     : Requested  initialize [chainDealId:0xe839d6f70530c18c7b367d233f66ceb437075a1e6fa341d79a50e0a6c849df91, taskIndex:0, waitingTxCount:0]
2019-02-07 10:03:08.649 ERROR 1 --- [pool-3-thread-1] com.iexec.common.chain.ChainUtils        : Transaction log received but blockNumber and txHash were both null inside [chainTaskId:0x03c34103ace93b3cbd84407fa94c8ce712652b340f55d4cd15770ac4de6fb0eb, action:Log{removed=false, logIndex='null', transactionIndex='null', transactionHash='null', blockHash='null', blockNumber='null', address='0x759b25b358b9f9c18812a69c0a9cf8b5a11c2e2d', data='0x', type='pending', topics=[0x252992fb0468d68d6a5784ec03214f0d0a362083f2d7ebd157af43b017a22e06, 0x03c34103ace93b3cbd84407fa94c8ce712652b340f55d4cd15770ac4de6fb0eb, 0x0000000000000000000000008a587620b9cd88171ccacbc561cec7a5ec5b109a]} receiptLog:0, lastBlock:{}]
2019-02-07 10:03:08.653  INFO 1 --- [pool-3-thread-1] com.iexec.core.chain.IexecHubService     : Initialized [chainTaskId:0x03c34103ace93b3cbd84407fa94c8ce712652b340f55d4cd15770ac4de6fb0eb, chainDealId:0xe839d6f70530c18c7b367d233f66ceb437075a1e6fa341d79a50e0a6c849df91, taskIndex:0, gasUsed:189527]```

**Comment by Zied-Guesmi** [comment]
Should we consider it as a successful tx or not ?

**Comment by jeremyjams** [comment]
Transaction did happen but with delay
https://kovan.etherscan.io/tx/0x93da08cc3a7e7eab884ba212a91730665e6a8cca96c76f8f0a9d05b404446c2b

We receiving a tx receipt whereas tx is still pending ..

**Comment by Ugo** [comment]
Fixed in the PR #199 .
